src/main/public/js/double-click-preventer.js

```diff
@@ -0,0 +1,12 @@
+(function () {
+  document.addEventListener('DOMContentLoaded', function (event) {
+
+    var submitInputTypeList = document.querySelectorAll('input[type=submit]')
+      .forEach(function (item) {
+        item.addEventListener('dblclick', function (event) {
```

Comment:
this doesn't address the intent of the story which is stopping multiple clicks / a click then a pause then another one,
also have you shown how pay does it to UX?